src/main/java/summarizer/team/ExtendedTeamEquipmentSummarizer.java

```diff
@@ -0,0 +1,44 @@
+package summarizer.team;
+
+import equipment.IEquipmentType;
+import equipment.describer.EquipmentDescriber;
+import team.ITeamMember;
+import equipment.AbstractEquipment;
+import summarizer.SummarizerUtils;
+
+import java.util.List;
+import java.util.Map;
+
+public class ExtendedTeamEquipmentSummarizer implements ITeamEquipmentSummarizer {
+
+    @Override
+    public TeamSummarizerType getSummaryType() {
+        return TeamSummarizerType.ExtendedTeamEquipmentSummarizer;
+    }
+
+    @Override
+    public String getTeamEquipmentSummary(Map<ITeamMember, List<AbstractEquipment>> teamEquipments) {
```

Comment:
why is this a `map` instead of a `list`? each `ITeamMember` already contains all of his associated equipment